### ISSUE TITLE
Pager#isInitialized is never true

### ISSUE THREAD

**Issue by NeverwinterMoon** [fermoya/SwiftUIPager#17]
I am not sure if it affects anything, but 

```swift
/// `true` if `Pager` already appeared in the screen at least once
@State var isInitialized = false
```

And:
```swift
.onAppear(perform: {
  if !self.isInitialized { // <- if false
    self.page = self.initialPage
    self.isInitialized = false // < - set to false
  }
})
```

Thus, `isInitialized` is never set to `true`

**Comment by fermoya** [comment]
Hi @NeverwinterMoon ,

It would affect if the `Pager` gets redrawn, indeed. 

Thanks for the input

**Comment by fermoya** [comment]
Hi @NeverwinterMoon,

I'm gonna close this as I've already released 1.3.1-beta.1 that contains this change.

Thanks
